src/frame/js/components/action.jsx

```diff
@@ -140,7 +140,8 @@ export class ActionComponent extends Component {
                                        currency={ currency.toUpperCase() }
                                        name={ stripeAccount.appName }
                                        image={ stripeAccount.iconUrl }
-                                       closed={ this.onStripeClose }>
+                                       closed={ this.onStripeClose }
+                                       executionContext={ parent }>
```

Comment:
`parent` is something available when inside an iframe. It refers to the context of the parent window.